Add event trackings when the "Launch paid campaign" buttons are clicked

eason9487 opened the pull request:
### Changes proposed in this Pull Request:

Closes #699 

- Add `launch_paid_campaign_button_click ` event trackings when "Launch paid campaign" buttons are clicked

### Screenshots:

![2021-06-01 14 26 39](https://user-images.githubusercontent.com/17420811/120278220-97814f00-c2e7-11eb-985d-bed4699f03fa.png)

### Detailed test instructions:

1. Open the DevTool console and run `localStorage.setItem( 'debug', 'wc-admin:*' );` to enable tracking debugging mode
2. Go to the Ads setup flow or paid campaign creation page
3. Finish the form and click the "Launch paid campaign" button to add a new paid campaign
4. The `gla_launch_paid_campaign_button_click` event should be logged on the DevTool console with respective `audience` and `budget`


### Changelog Note:

> Add event trackings when the "Launch paid campaign" buttons are clicked

eason9487 committed:
Add launch_paid_campaign_clicked event trackings when "Launch paid campaign" buttons are clicked

eason9487 committed:
Update tracking README for the launch_paid_campaign_clicked event

eason9487 committed:
Rename launch_paid_campaign_clicked to launch_paid_campaign_button_click

mikkamp reviewed (approved):
Working as expected when testing
![image](https://user-images.githubusercontent.com/11388669/120605614-2d32f080-c446-11eb-8d6a-0c6c78d2d988.png)

It just needs the conflict resolved for the readme and swap the two descriptions.

eason9487 committed:
Fix the incorrect doc descriptions of launch_paid_campaign_button_click event

Address https://github.com/woocommerce/google-listings-and-ads/pull/711#pullrequestreview-674962551
